cli/command/service/opts.go

```diff
@@ -182,7 +182,7 @@ func (o *SecretOpt) Set(value string) error {
 
 		value := parts[1]
 		switch key {
-		case "source":
+		case "source", "src":
```

Comment:
Yup, I see the issue. Difference with volumes is of course that `src` _does_ make sense if it's a bind-mount.